coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Coverage files made it back in during the conflict resolutions and need to be removed